plugin/clone/src/clone_client.cc

```diff
@@ -1058,6 +1079,25 @@ int Client::validate_remote_params() {
       continue;
     }
 
+    /* Plugin is not installed in recipient but active on donor. Check if this
+    plugin can be exempted from matching with donor */
+    if (clone_exclude_plugins_match != nullptr) {
+      std::string plugin_str(plugin_name);
+      std::transform(plugin_str.begin(), plugin_str.end(), plugin_str.begin(),
+                     ::tolower);
+
+      if (std::find(excl_plugins_vec.begin(), excl_plugins_vec.end(),
+                    plugin_str) != excl_plugins_vec.end()) {
+        char info_mesg[128];
+        snprintf(info_mesg, 128,
+                 "Plugin %s is not installed on recipient but ignored due to "
```

Comment:
The only reason the plugin not being installed is an issue is because it's on the source? If so say so as the read er of the error message won't realise why the plugin is being installed.

e.g. `"Ignoring plugin %s installed on the source server but not the recipient.  The mismatch is ignored due to 'clone_exclude_plugins_match'"`

I think that something like this slightly different comment makes the DBA aware of (i) why there's a mismatch and why it's excluded in this specific case.